uc-crux-llvm/src/UCCrux/LLVM/Overrides/Skip.hs

```diff
@@ -162,37 +369,98 @@ createSkipOverride ::
   IORef (Set SkipOverrideName) ->
   -- | Annotations of created values
   IORef (Map (Some (What4.SymAnnotation sym)) (Some (TypedSelector m arch argTypes))) ->
+  -- | What data should get clobbered by this override?
+  ClobberSpecs m ->
+  -- | Constraints on the return value
   Maybe (ConstrainedTypedValue m) ->
-  L.Declare ->
+  -- | Function to be overridden
   FuncSymbol m ->
-  Maybe (PolymorphicLLVMOverride arch (personality sym) sym)
-createSkipOverride modCtx sym usedRef annotationRef postcondition decl funcSym =
+  Maybe (Either (ClobberSpecError m) (PolymorphicLLVMOverride arch (personality sym) sym))
```

Comment:
I'm a bit surprised to see an `Either` inside of a `Maybe`. Do we need both layers?

Reply:
Fixed.